delta/sourcing-psql/src/main/scala/ch/epfl/bluebrain/nexus/delta/sourcing/stream/ProjectionStore.scala

```diff
@@ -185,12 +187,11 @@ object ProjectionStore {
       override def saveFailedElems(
           metadata: ProjectionMetadata,
           failures: List[FailedElem]
-      ): UIO[Unit] =
-        failures
-          .traverse(elem => saveFailedElem(metadata, elem))
-          .transact(xas.write)
-          .void
-          .hideErrors
+      ): UIO[Unit] = {
+        val log  = UIO(logger.debug(s"[${metadata.name}] Saving ${failures.length} failed elems."))
```

Comment:
Seems to only repeat the previous one

Reply:
I kept the one inside the `saveFailedElems` function.